python/src/worker/tasks/differential_expression.py

```diff
@@ -112,12 +113,19 @@ def compute(self):
         if len(second_cell_set) == 0:
             raise Exception("No cells id fullfills the 2nd cell set.")
 
-        # create request from the corresponding marked cells
         request = {
             "baseCells": [int(x) for x in first_cell_set],
             "backgroundCells": [int(x) for x in second_cell_set],
+            "pagination": self.pagination,
         }
 
+        if "filters" in self.pagination:
+            gene_filter = self.pagination["filters"][0]["expression"]
```

Comment:
I would make this a foreach instead of accessing the first element

Reply:
can you have more than 1 element under filters? And if so, why aren't we looking at all of them? If not, then why is `filters` entry a list, rather than an an object with 1 entry in it?